src/views/Profile/ProfileCreation/Mint.tsx

```diff
@@ -14,16 +14,18 @@ import useProfileCreation from './contexts/hook'
 
 const starterBunnyIds = [5, 6, 7, 8, 9]
 const nfts = nftList.filter((nft) => starterBunnyIds.includes(nft.bunnyId))
-const minimumCakeBalance = 4
+const cakeCostToMint = 4
+const minimumCakeBalanceToMint = new BigNumber(cakeCostToMint).multipliedBy(new BigNumber(10).pow(18))
```

Comment:
I plan on checking if we can get these values from the contract so we don't have to explicitly set them here and in Username.